templates/jinja_macros/product_listing.html

```diff
@@ -10,47 +10,127 @@
                     <br/>
                 </b>
             {% endfor %}
+{%- endmacro -%}
+{%- macro modal(product) -%}
+    <script>
+        // Script for closing the product information modal
+        document.addEventListener('DOMContentLoaded', () => {
+            function closeModal($el) {
+                $el.classList.remove('is-active');
+            }
+
+            (document.querySelectorAll('.modal-background, .modal-close, .modal-card-head .delete') || []).forEach(($close) => {
+                const $target = $close.closest('.modal');
+                $close.addEventListener('click', () => {
+                    closeModal($target);
+                });
+            });
+        });
+    </script>
+    <div class="modal" id="{{ product.name }}_modal">
+        <div class="modal-background"></div>
+        <div class="modal-card"style="border-radius: 1em;">
+            <header class="modal-card-head">
+                <p class="modal-card-title">Product information</p>
+                <button class="delete is-large" aria-label="close"></button>
+            </header>
+            <section class="modal-card-body">
+                <article class="media">
+                    <figure class="media-left">
+                        <img class="center"
+                             src="{{ product.image }}"
+                             alt="Image for : {{ product.name }}"
+                             style="max-height:50em;
+                                    max-width:12em;
+                                    height:auto;
+                                    width:auto">
+                    </figure>
+                    <div class="media-content">
+                        <div class="content">
+                            <div class="column">
+                                <div class="content">
+                                    <div class="level is-mobile">
+                                        <!-- These should be fetched from the db -->

```

Comment:
Remove this level until we actually fetch the attributes from db

Reply:
The level has been removed.